client/boot/index.js

```diff
@@ -51,6 +52,9 @@ function init() {
 
 	debug( 'Starting Calypso. Let\'s do this.' );
 
+	// prune sync-handler records
+	syncHandler.pruneRecordsFrom( '30 minutes ago' );
```

Comment:
Is this just for demonstration? Why would we prune records that are only 30 minutes old? The purpose of invalidation is to make sure we don't outgrow our storage needs, I think we have quite a bit more room than this.